### ISSUE TITLE
Error: Symbol error not found, cannot apply R_X86_64_JUMP_SLOT

### ISSUE THREAD

**Issue by n3oney** [ptitSeb/box64#1228]
As in the title, I get this error:
`Error: Symbol error not found, cannot apply R_X86_64_JUMP_SLOT @0x40af48 (0x4022c6) in /nix/store/f2vs29jibd7lwxyj35r9h87h6brgdysz-hello-2.12.1/bin/hello`

I'm just trying to run the hello program compiled for x86_64-linux on aarch64-linux

Full output:

```
❯ box64 hello 
Dynarec for ARM64, with extension: ASIMD AES CRC32 PMULL ATOMICS SHA1 SHA2 USCAT FLAGM FLAGM2 FRINT PageSize:16384 Running on Icestorm-M1 Firestorm-M1 with 8 Cores
Params database has 14 entries
Box64 with Dynarec v0.2.7 nogit built on Jan  1 1980 00:00:00
BOX64: Detected 48bits at least of address space
Counted 80 Env var
BOX64 LIB PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
BOX64 BIN PATH: ./:bin/:/nix/store/f2vs29jibd7lwxyj35r9h87h6brgdysz-hello-2.12.1/bin/:/run/wrappers/bin/:/home/neoney/.nix-profile/bin/:/nix/profile/bin/:/home/neoney/.local/state/nix/profile/bin/:/etc/profiles/per-user/neoney/bin/:/nix/var/nix/profiles/default/bin/:/run/current-system/sw/bin/:/nix/store/5ray4n6gr86gfvlnjqxf7yihdwq8pmrj-gcc-wrapper-13.2.0/bin/:/nix/store/vgp3k47jqi8i867rc0y9j20ww8ql7y6y-binutils-wrapper-2.40/bin/:/nix/store/rcbwdnd73xsvy87nbcbmc9pdbdr3i3ay-pciutils-3.10.0/bin/
Looking for hello
Rename process to "hello"
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux-x86-64.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libdl.so.2
Using native(wrapped) libutil.so.1
Using native(wrapped) libresolv.so.2
Using native(wrapped) librt.so.1
Using native(wrapped) libbsd.so.0
Error: Symbol error not found, cannot apply R_X86_64_JUMP_SLOT @0x40af48 (0x4022c6) in /nix/store/f2vs29jibd7lwxyj35r9h87h6brgdysz-hello-2.12.1/bin/hello
```

I'm using box64 built from the latest commit (the one 5 minutes ago) with -D M1=1, as I'm on M1

**Comment by n3oney** [comment]
When trying to run nodejs, I seem to be getting the same for

```
Error: Symbol __isoc23_strtoll not found, cannot apply R_X86_64_JUMP_SLOT @0x2d5da88 (0xa956b6) in /nix/store/d6lkbndr98lcn8spbqxfq52f2ldvqhks-nodejs-20.11.0/bin/node
Error: Symbol __isoc23_strtoull not found, cannot apply R_X86_64_JUMP_SLOT @0x2d5e970 (0xa97486) in /nix/store/d6lkbndr98lcn8spbqxfq52f2ldvqhks-nodejs-20.11.0/bin/node
```

**Comment by ValdikSS** [comment]
Related: https://github.com/ptitSeb/box86/issues/923

The application is aimed at specific userspace, it's hard to follow all updates from glibc and other libraries.

**Comment by losehu** [comment]
i am M1 too ,with asahi linux,i couldn't use box64 too
